Ginger/GingerCoreCommon/Repository/BusinessFlowLib/ActivitiesGroup.cs

```diff
@@ -87,6 +87,11 @@ public string Description
             }
         }
 
+        public override string ToString()
```

Comment:
for what it is needed? for pulling the name for dragging window show?

Reply:
Yes Meni